lib/firenest/pub_sub.ex

```diff
@@ -269,8 +269,7 @@ defmodule Firenest.PubSub.Supervisor do
   end
 
   def init({pubsub, topology, options}) do
-    partitions =
-      options[:partitions] || System.schedulers_online() |> Kernel./(4) |> Float.ceil() |> trunc()
+    partitions = options[:partitions] || ceil(System.schedulers_online() / 4)
```

Comment:
Just came across this, cool project! Doesn't look like there has been much movement recently so I figured I'd just point out that the tests are failing due to this line, `ceil/1` doesn't exist, perhaps you meant to leave it as `Float.ceil/1`.